docs/documentation/stories/include-bootstrap.md

```diff
@@ -22,11 +22,11 @@ With the new project created and ready you will next need to install bootstrap t
 Using the `--save` option the dependency will be saved in package.json
 
 ```sh
-# version 3.x
```

Comment:
Do we typically add instructions for adding old packages to projects? I'd suggest we remove the v.3 installation notes altogether.

Reply:
Yes. You are right. I updated the PR.